python_modules/dagster/dagster/_cli/run.py

```diff
@@ -70,3 +71,70 @@ def run_wipe_command(force):
         click.echo("Deleted all run history and event logs.")
     else:
         raise click.ClickException("Exiting without deleting all run history and event logs.")
+
+
+@run_cli.command(name="migrate", help="Migrate the run history from one repository to another.")
```

Comment:
What do you think about calling it "migrate-repository" in case there are other kinds of migrations we want to enable in the future?